content/documentation/multiformat-output.md

```diff
@@ -1,12 +1,218 @@
+
+1. Output your PDF and e-book files following the directions above, if you are going to include them as part of your published project
+2. Update the baseURL in config/site.yml to match the URL where the site will ultimately be hosted
+3. Output your site following the directions above
+4. Upload the contents of the site folder to your web host based on the directions they provide
+
+If you do not already have a web server or hosting plan, we’ve found that [Netlify](https://www.netlify.com/?_ga=2.127713933.1882208577.1616011630-1141534382.1615320952) offers a couple excellent options.
+
+### Manual Deploy with Netlify
+
+Manual deploy is ideal if you have a small site or want to run a quick preview. Keep in mind, with this option, you will need to go through the process of rebuilding the site, compressing files, and reuploading them to Netlify each time you make an update, which may be burdensome if you have lots of images or larger files. For continuous deployment please see [*Continuous Deploy from GitHub*](#continuous-deploys-from-github).
+
+1. Navigate to your project and compress the `site` folder.
+
+2. Go to Netlify Drop: [https://app.netlify.com/drop](https://app.netlify.com/drop). Make sure you are logged in to your account and then drag-and-drop your compressed `site` folder into the indicated area.
+
+3. You will be given a default URL to preview your project. Rename this URL by navigating to “Site settings” and changing the site name. (You also have the option to buy a domain or set-up a domain you already own.)
+
+If you make further edits to your project and would like to preview them you will need to repeat this process.
+
+1. Delete the old compressed `site` folder.
+
+2. Run `quire site` again (your files will be automatically overwritten.)
+
+3. Compress the newly updated `site` folder.
+
+4. In Netlify, navigate to “Deploys” at the top of the page. You will see a blank space that reads, “Need to update your site." Simply drag-and-drop your new compressed `site` folder here and your link will be automatically updated.
+
+### Continuous Deploy with Netlify
+
+By linking GitHub directly to your Netlify account, any time you merge changes in Github, your preview link will be automatically updated. This process requires a few extra steps to get set-up, but will save time in the long run.
+
+1. If you haven't already, create a repository for your project on GitHub. You can find directions in the [*GitHub*](/documentation/gitub) section of our documentation.
+
+2. To proceed with deployment, you will need to add two files to your repo: `netlify.toml` and `package.json`. Download them [here](/downloads/site-deploy.zip), add them to your repo, and merge changes before proceeding.
+
+3. Log in to [Netlify](https://app.netlify.com/). On the “Team Overview” page, click the button that says “New Site from Git.”
+
+4. Next you'll want to “Connect to Git Provider.” Choose either GitHub or GitLab.
```

Comment:
Oh, here's a GitLab reference. And Bitbucket works too, tut I think just one note at the top that all three services work and then consistently using GitHub throughout would be fine.